src/main/resources/liquibase/workbench_changelog/20_1_0.xml

```diff
@@ -51,4 +51,43 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v20.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			  <sqlCheck expectedResult="0">
+                SELECT count(*) FROM workbench_tool WHERE name = 'crop_settings_manager'
+            </sqlCheck>
+        </preConditions>
+		<comment>
+			Add crop settings manager sidebar item.
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO workbench_tool (`name`, `group_name`, `title`, `version`, `tool_type`, `path`, `parameter`, `user_tool`)
+			VALUES ('crop_settings_manager', 'crop_settings_manager', 'Manage Crop Settings', '1.0', 'WEB', '/ibpworkbench/controller/jhipster#crop-settings-manager', '', 0);
```

Comment:
@cuenyad are not we using the version of the release here? should it be 20.1 instead? the only one with 1.0 of the new modules is search_germplasm, in the rest of the cases we are using the minor version number